### PR TITLE
chore: add license

### PR DESCRIPTION
**Pull request by augustobmoura**
[FX-2619](https://toptal-core.atlassian.net/browse/FX-2619)

### Description

Add license to both the project and generated packages

### How to test

- Checkout the project locally, run `yarn build` and check if all `dist-packages` have a LICENSE file in it

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: bin/build.js [hunk 1]
```diff
@@ -33,6 +39,7 @@ const compile = function(tsConfig, packageJson, packageRootDir) {
 
   copyPackageJson(packageRootDir)
   copyReadme(packageRootDir)
+  copyLicense(packageRootDir)
```
Comment (dmaklygin) [inline bin/build.js:42]: 👍

File: bin/utils/copy-license.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const path = require('path')
+const fs = require('fs')
+
+const { log } = require('./log')
+const { BUILD_FOLDER } = require('./constants')
+
+const copyLicense = packageRootDir => {
```
Comment (dmaklygin) [inline bin/utils/copy-license.js:7]: `packageRootDir` is the same folder where root package.json is kept?

File: bin/utils/copy-license.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  log('')
+  log(
+    `Copying LICENSE file to the build folder "${BUILD_FOLDER}" of the package: ${packageRootDir}`,
+    'magenta'
+  )
+
+  const licenseFile = path.resolve(__dirname, '../../LICENSE')
```
Comment (dmaklygin) [inline bin/utils/copy-license.js:14]: can we reuse `packageRootDir` ?

**Review by dmaklygin — APPROVED** [review]

**Review by TomasSlama — APPROVED** [review]

**Review by OndrejTuma — APPROVED** [review]
LTGM 👍

File: packages/picasso-codemod/bin/build.sh [hunk 4]
```diff
@@ -6,3 +6,4 @@ cp -R ./bin ./dist-package/bin
 cp -R ./src ./dist-package
 cp ./README.md ./dist-package/README.md
 cp ./package.json ./dist-package/package.json
+cp ../../LICENSE ./dist-package/LICENSE
```
Comment (denieler) [inline packages/picasso-codemod/bin/build.sh:9]: why do we need this for codemod?

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,18 @@
+Copyright © 2021-2022 Toptal, LLC
```
Comment (denieler) [inline LICENSE:1]:
```suggestion
MIT License

Copyright © 2021-2022 Toptal, LLC
```

Comment (denieler) [reply LICENSE:1]: https://github.com/facebook/lexical/blob/main/LICENSE

File: packages/picasso-codemod/bin/build.sh (hunk 4 above)
Comment (augustobmoura) [reply packages/picasso-codemod/bin/build.sh:9]: Sorry, can you be more clear? Why would it not need a license? `picasso-codemod` is not an open source project under MIT?

File: bin/utils/copy-license.js (hunk 2 above)
Comment (augustobmoura) [reply bin/utils/copy-license.js:7]: It's the directory for the package (eg `packages/picasso`, `pcakages/picasso-codemod`), it's the same naming as `copy-package-json.js`

Comment (augustobmoura) [reply bin/utils/copy-license.js:7]: I will rename it to a more clear name

File: bin/utils/copy-license.js (hunk 3 above)
Comment (augustobmoura) [reply bin/utils/copy-license.js:14]: Yep, can do. Assuming all packages are on the directory `packages` it should be the same thing

**Review by denieler — APPROVED** [review]

**Pull request merged**